provisioning/Prospero_update_to_version.adoc

```diff
@@ -0,0 +1,177 @@
+---
+categories:
+ - provisioning
+stability-level: default
+# Specify the Feature Development tracker issue for the feature.
+# This must be an issue tracked in https://github.com/orgs/wildfly/projects/7/views/1.
+# To create a Feature Development tracker issue, go to  https://github.com/wildfly/wildfly-proposals/issues/new/choose
+# and select 'Feature Development'
+issue: https://github.com/wildfly/wildfly-proposals/issues/705
+# Provide the github ids of the members of the feature team, organized by role.
+# Provide a single id for the 'assignee' role. Use a yaml list for the 'sme' and
+# 'outside-perspective' roles, even if there is only one person in a role.
+feature-team:
+ developer: Bartosz Spyrko-Smietanko
+ sme:
```

Comment:
@spyrkob I can be the `sme` for this feature if you're ok with it?

Reply:
Sure, that'd be great, thank you!